independent-publisher-connectors/OpenAI/OpenAI.swagger.json

```diff
@@ -0,0 +1,261 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI",
+    "description": "Connect to the OpenAI API and use the Power of GPT3, API key must be entered as \"Bearer YOUR_API_KEY\"",
+    "version": "1.0"
+  },
+  "host": "api.openai.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/engines/{engine}/completions": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@PowerRobin Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Changed it to 200 and provided a description.
Sorry, looked at a bad example